index.html

```diff
@@ -0,0 +1,45 @@
+<DOCTYPE html>
+  <html lang="en">
+    <head>
+      <link rel="stylesheet" href="styles.css" >
+      <title>Landing Page</title>
+    
+    </head>
+    
+
+    <body>
+      <header id="header">
+        <img id="img-header" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png"/>
+        <nav id="nav-link">
+          <ul>
+            <li><a href="#">Features</a></li>
+            <li><a href="#">How It Works</a></li>
+            <li><a href="#">Pricing</a></li>
+          </ul>
+        </nav>
+      </header>
+      <h1>Handcrafted, home-made masterpieces</h1>
+      <form action="https://www.freecodecamp.com/email-submit" id="form" > <input id="email" placeholder="Enter your email adress" required type="email" >
+    <input id="submit" type="submit">
+      </form>
+      <div>
+        <h2>Premium Materials</h2>
+        <p>Our trombones use the shiniest brass which is sourced locally. This will increase the longevity of your purchase. </p>
+        <h2>
+          Fast Shipping</h2>
+        <p>We make sure you recieve your trombone as soon as we have finished making it. We also provide free returns if you are not satisfied. </p>
+        <h2>
+          Quality Assurance</h2>
+        <p>For every purchase you make, we will ensure there are no damages or faults and we will check and test the pitch of your instrument. </p>
+      </div>
+      <div>
+        <iframe id="video" src="https://www.youtube-nocookie.com/embed/y8Yv4pnO7qc?rel=0&controls=0&showinfo=0" controls></iframe>
+      </div>
+      <footer>
+        <h3>Privacy</h3>
+        <h3>Terms</h3>
+        <h3>Contact</h3>
+        <p>Copyright 2016, Original Trombones</p>
+      </footer>
+    </body>
+  </html>
```

Comment:
nice work with the Html file, just need to use good practices like goog indentation and delete white space.  [read more here.](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)